### PR TITLE
Zigbee: Fix building light bulb on nrf21540dk_nrf52840

### PR DESCRIPTION
**Pull request by jangalda-nsc**
Add missing configuration for nrf21540dk_nrf52840 board.
Signed-off-by: Jan Gałda <jan.galda@nordicsemi.no>

### CODE REVIEW

File: samples/zigbee/light_bulb/boards/nrf21540dk_nrf52840.overlay [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021 Nordic Semiconductor ASA
+ * Copyright (c) 2020 Nordic Semiconductor ASA
```
Comment (rlubos) [inline samples/zigbee/light_bulb/boards/nrf21540dk_nrf52840.overlay:2]: This doesn't seem like an intended change.

Comment (jangalda-nsc) [reply samples/zigbee/light_bulb/boards/nrf21540dk_nrf52840.overlay]: Thanks, fixed

**Review by rlubos — APPROVED** [review]

**Review by sebastiandraus — APPROVED** [review]
LGTM

**Review by maciekfabia — APPROVED** [review]
LGTM

**Review by wbober — APPROVED** [review]

**Pull request merged**